cypress/support/custom_commands/login.js

```diff
@@ -2,80 +2,83 @@ const emailInputFieldElement = '[data-testid="username-email"]'
 const passwordInputFieldElement = '[data-testid="password-email"]'
 const submitButton = '[data-testid="submit-login-email"]'
 const nbcLoginWithEmailOptionButton = '[data-testid="submit-cloud-site"]'
+const defaultLoginViaExternalBroker = '[data-testid="submit-oauth-login"], [data-provider="oauth"]'
 const initials = '[data-testid="initials"]'
 const languageSelection = '[id="selected-language"]'
 const languageDe = '[data-language="de"]'
 
+const externalUsernameInputFieldElement = '[id="Username"]'
+const externalPasswordInputFieldElement = '[id="Password"]'
+
 Cypress.Commands.add('login', (username, environment) => {
   cy.session([username, environment], () => {
     const env = Cypress.env()
-    let link
-    let environmentUpperCased = environment.toUpperCase()
-    if (environmentUpperCased === 'BRB') {
-      link = Cypress.config('baseUrl', env[environmentUpperCased])
-      cy.visit(link)
-    } else if (environmentUpperCased === 'NBC') {
-      link = Cypress.config('baseUrl', env[environmentUpperCased])
-      cy.visit(link)
+    const environmentUpperCased = environment.toUpperCase()
+    const link = Cypress.config('baseUrl', env[environmentUpperCased])
+    cy.visit(link)
+    if (environmentUpperCased === 'NBC') {
       cy.get(nbcLoginWithEmailOptionButton).eq(1).click()
-    } else if (environmentUpperCased === 'DEFAULT') {
-      link = Cypress.config('baseUrl', env[environmentUpperCased])
-      cy.visit(link)
+    }
+    if (environmentUpperCased === 'DEFAULT') {
+      cy.visit('/login');
```

Comment:
'login' can be removed from here because there is '/login' already appended in the default URL in cypress.env.json
 so in this line it should be-> cy.visit('/');

Reply:
as discussed, the redirect on DEFAULT points to 'login-instances' rather then 'login'. Hence the visit to login.

Reply:
yes thats right, thats why we already added '/login' in the cypress.env.json file, tests failing if add two time /login' of course